src/executables/uber-time.js

```diff
@@ -9,12 +9,22 @@ import CommandExecutionService from '../services/CommandExecutionService';
 const service = new CommandExecutionService();
 
 program
-  .arguments('<address>')
-  .action((address) => {
-    try {
-      service.executeTimeEstimates(address).then(table => console.log(table));
-    } catch (Error) {
-      console.error('Could not get time estimates');
-    }
-  })
+  .usage('<address>')
+  .description('Get Time-To-Pickup Estimates')
   .parse(process.argv);
+
+const address = program.args.toString();
```

Comment:
I'm not sure this deals with the `trim` case...?

![image](https://user-images.githubusercontent.com/8136030/27619893-bdaaaaa6-5b93-11e7-9481-969aae4825eb.png)

vs.

![image](https://user-images.githubusercontent.com/8136030/27619901-c887b04a-5b93-11e7-831a-f452f4f57b7e.png)